src/engines/google/google.go

```diff
@@ -9,12 +9,13 @@ import (
 
```

Comment:
It's bad practice to change one feature in another feature's PR (at least mention it in the description or even better TITLE of the PR). Optimal solution would be a separate PR.